vmware_dvs/agent/firewalls/vcenter_firewall.py

```diff
@@ -83,9 +83,16 @@ def get_update_tasks(self, number=5):
         return None, []
 
     def get_remove_tasks(self):
-        ret = self.remove_store
-        self.remove_store = []
-        return ret
+        ret = []
+        for dvs, tasks in self.remove_store.iteritems():
```

Comment:
It's better to use six.iteritems(self.remove_store) or self.remove_store.items() for py3 code compatibility